src/components/atom/input/_placeholders.scss

```diff
@@ -2,15 +2,15 @@
   background: $c-white;
   border: $bdw-s solid $c-gray-light;
   box-sizing: border-box;
-  font-size: $fz-base;
+  font-size: $fz-atom-input;
   min-height: $h-atom-input--m;
   padding-left: $pl-atom-input;
   padding-right: $pr-atom-input;
   width: 100%;
 }
 
 %sui-atom-input-input-focus {
-  border: $bdw-s solid $c-primary;
-  box-shadow: $bxsh-focus;
+  border: $b-atom-input;
```

Comment:
border abbreviation should be `bd` instead of `b`

Reply:
You're right. I guess we should open a new PR to fix it

Reply:
I open a PR to fix it